example-projects/game-processor/asyncapi.yaml

```diff
@@ -0,0 +1,66 @@
+asyncapi: 2.0.0
+info: 
+  title: "Processor"
+  version: "0.0.1"  
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+servers: 
+  local: 
+    url: "0.0.0.0:1883"
+    description: "Development server",
+    protocol: "mqtt"
+defaultContentType: application/json
+channels: 
+  game/server/{serverId}/events/player/{playerId}/item/{itemId}/pickup:
+    description: Channel used when a player picks up an item in-game
+    parameters:
+      serverId: 
+        $ref: "./components/Parameters.yaml#/serverId"
```

Comment:
why not ./AsyncAPI/components/Parameters.yaml#/serverId

Reply:
Wups! Copy paste mistake, gonna fix it tomorrow 🙂

Reply:
Fixed 👍